src/app/backend/handler/apihandler.go

```diff
@@ -136,7 +135,7 @@ func CreateHTTPAPIHandler(client *clientK8s.Clientset, heapsterClient client.Hea
 	verber := common.NewResourceVerber(client.Core().RESTClient(),
```

Comment:
I think the RESTClients here won't forward the bearer token now. I think that this is a serious security issue if you want to use this Bearer token feature in some enforced way